frontend/app/model/Eventbrite.scala

```diff
@@ -187,6 +187,7 @@ object Eventbrite {
       case t: InternalTicketing => t
     }
 
+    val ticketsNotSold = capacity - ticket_classes.map(_.quantity_sold).sum
```

Comment:
This variable definitely doesn't belong here, it should be on the `InternalTicketing` object - it can only be meaningful if we're using internal ticketing (ie not selling through Picturehouses or whatever).